exercises/matthew_cumber/ex02_oo_basics/src/Shape.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <string>
+
+/*
+ * Abstract class for all shapes for common members and methods
+ * m_type : The type of shape determined by derived class
+ * m_numSides : The number of sides the shape has determined by derived class
+ */
+class Shape
+{
+public:
+  virtual ~Shape() = default;
```

Comment:
We don't need to include the default destructor here as it should already be set to default